### PR TITLE
[DOCS] Update internal link to external

### PR DESCRIPTION
**Pull request by EamonnTP**
To accommodate including the **Install and run** content in the Observability docs, this PR updates an internal link to external.

Related PR: elastic/observability-docs#16

### CODE REVIEW

File: docs/guide/install-and-run.asciidoc [hunk 1]
```diff
@@ -142,7 +142,7 @@ and configuring it with the address of your APM Server, a secret token (if neces
 
 |=======================================================================
 
-TIP: Check the <<agent-server-compatibility,Agent/Server compatibility matrix>> to ensure you're using agents that are compatible with your version of Elasticsearch.
+TIP: Check the {apm-py-ref-v}/agent-server-compatibility.html[Agent/Server compatibility matrix] to ensure you're using agents that are compatible with your version of Elasticsearch.
```
Comment (bmorelli25) [inline docs/guide/install-and-run.asciidoc:145]:
```suggestion
TIP: Check the {apm-overview-ref-v}/agent-server-compatibility.html[Agent/Server compatibility matrix] to ensure you're using agents that are compatible with your version of Elasticsearch.
```

Comment (bmorelli25) [reply docs/guide/install-and-run.asciidoc:145]: Wrong book

Comment (bmorelli25) [reply docs/guide/install-and-run.asciidoc:145]: Once you make this update and the tests pass, feel free to merge this and open the backports. If for some reason you can't merge, most of the Server devs are in your time zone. Just post in #apm-server

Comment (EamonnTP) [reply docs/guide/install-and-run.asciidoc]: Thanks

**Pull request merged**